templates/body_template.hbs

```diff
@@ -16,17 +16,20 @@
 + ({{name}}*)create:(void(^)({{name}}Builder *builder))builderBlock
 {
     {{name}}Builder *builder = [{{name}}Builder new];
-    if(builderBlock)
-    {
+    {{#props}}
+    {{#if default}}
+    builder.{{name}} = {{default}};
+    {{/if}}
+    {{/props}}
+    if (builderBlock) {
         builderBlock(builder);
     }
     return [[self alloc] initWithBuilder:builder modelVersion:1];
 }
 
 - (instancetype)init
 {
-    {{name}}Builder *builder = [{{name}}Builder new];
-    return [[[self class] alloc] initWithBuilder:builder modelVersion:1];
+    return [self create:NULL];
```

Comment:
Here `create:`  it's a class method (static). So it will crash with `unrecognized selector sent to instance`

Reply:
fixed